src/controllers/Votes.js

```diff
@@ -0,0 +1,35 @@
+import db from '../models/index';
+
+
+class vote {
+  static async create(req, res) {
+    const createQuery = `INSERT INTO
+    votes(office, candidate, voter, createdOn)
+    VALUES($1, $2, $3, $4)
+    returning *`;
+
+    const values = [
+      req.body.office,
+      req.body.candidate,
+      req.user.id,
+      req.body.createdOn,
+    ];
+
+    try {
+      const { rows } = await db.query(createQuery, values);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          id: rows[0].id,
+          message: 'vote created',
```

Comment:
Same here according to the spec this should be your response object 
![image](https://user-images.githubusercontent.com/24937059/51975401-662bb680-2482-11e9-930b-efb661e5230f.png)